src/azurelinux/3.0/net9.0/crossdeps/amd64/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN tdnf update -y && \
         diffutils \
         icu \
         tar \
+        cpio \
```

Comment:
Please keep these in alphabetical order for each comment section. So this one should be placed after `awk`. Same applies to the other file.